clickhandlers.js

```diff
@@ -0,0 +1,11 @@
+var CarLot = (function(orginal) {
```

Comment:
This function is supposed to take two args. Revisit the requirements for clarity